packages/lib/models/BaseItem.test.ts

```diff
@@ -162,3 +162,34 @@ three line \\n no escape`)).toBe(0);
 		expect(await BaseItem.loadItemById(note.id)).toMatchObject({ id });
 	});
 });
+
+describe("isSystemPath - MC/DC", () => {
```

Comment:
Thank you for the pull request!

Feedback:
- Be sure that code [passes the linter](https://github.com/laurent22/joplin/blob/dev/readme/dev/coding_style.md#enforcing-rules-using-eslint).
- Be sure that the code passes the typechecker prior to committing.
- Usually, the `.test.ts` files should have a single `describe` block per file. 
- Test names should use English and usually should be in the form `it('should ...'`.
- Consider using the [`test.each` pattern](https://jestjs.io/docs/api#testeachtablename-fn-timeout) when there are many similar tests with different inputs/outputs.